app/templates/suppliers/details.html

```diff
@@ -170,16 +172,41 @@
 
 {% endcall %}
 
-{% if currently_applying_to %}
-  <p>You must complete your company details to make an application.</p>
-  {%
-    with
-    url = url_for(".framework_dashboard", framework_slug=currently_applying_to.slug),
-    text = "Return to your {} application".format(currently_applying_to.name),
-    bigger = false
-  %}
-    {% include "toolkit/secondary-action-link.html" %}
-  {% endwith %}
-{% endif %}
+  </div>
+
+  {% if not supplier.companyDetailsConfirmed %}
+    <div class='column-two-thirds'>
```

Comment:
Should separate `column-<some-width>` divs go in their own grid row if they're going to appear below other divs, like the one above? Asking cos I don't know.

Reply:
Good question that I don't really know the answer to - the frontend toolkit only has examples for single rows 🤔.   Feels like logically yes they probably should, though I have a feeling it "works" without, and we might have lots of places where they don't.  I'm going to make a new grid-row here though now you've mentioned it.